packages/core/src/codewhisperer/service/transformByQ/transformFileHandler.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { existsSync, writeFileSync } from 'fs'
```

Comment:
Question: We should strictly true to use `fs` and not `fs-extra` correct?

Reply:
Should use FileSystemCommon: https://github.com/aws/aws-toolkit-vscode/blob/897ef7d03adf117ea5873d3cbc396e394b25c800/packages/core/src/srcShared/fs.ts#L32